corrections to Connect to Box -- misspellings, missing words, formatting (#2391)
MicrosoftDocs/OfficeDocs-SharePoint: migration/mm-box-connection.md

> 1. From the [Migration page in the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=migrationCenter&modern) and select Box "Get started".

Reviewer: ```suggestion
1. From the [Migration page in the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=migrationCenter&modern), select the **Get started** box.
```

> 1. From the [Migration page in the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=migrationCenter&modern) and select Box "Get started".

Reviewer: ```suggestion
1. From the [Migration page in the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=migrationCenter&modern), select Box "Get started".
```

2. Select **Connect to Box**. 

> 3. Mover works in the background to scan Box, and copy data to the Microsoft 365.  Authorize Mover with your Microsoft credentials to connect your co-admin Box account. Click **Authorize Mover**.

Reviewer: ```suggestion
3. Mover works in the background to scan Box and to copy data to Microsoft 365.  Authorize Mover with your Microsoft credentials to connect your co-admin Box account. Select **Authorize Mover**.
```

4. Sign in with an account that has [admin permissions](/sharepoint/sharepoint-admin-role) for your organization.  
5. Sign in to Box with your co-admin account. This will connect your Box account to Microsoft 365.

> 6. Log in to grant access to Box. Click **Authorize.**

Reviewer: ```suggestion
6. Log in to grant access to Box. Select **Authorize.**
```

7. Install the Microsoft Box app in your Box account settings. This app will authorize content to be copied from your Box account to Microsoft 365. First copy the app client ID click copy.
8. Use that copied client ID to add the Microsoft app. Go you your Box account settings to authorize a new app in your Box account settings and enter the client ID when prompted.

> 9. In the Box screen -- click "Authorize new app".

Reviewer: ```suggestion
9. In the Box screen, select "Authorize new app".
```

> 10. Paste in the client ID you copied from the earlier screen. Enter the client id for application for which you would like to grant access.

Reviewer: ```suggestion
10. Paste in the client ID you copied from the earlier screen. Enter the client id for the application for which you would like to grant access.
```

> 11. An App Authorization page appears and lists the authorization details of the request. Click "Authorize".

Reviewer: ```suggestion
11. An App Authorization page appears and lists the authorization details of the request. Select "Authorize".
```

> 12. You are now connected to Box. Click Finish to close window.

Reviewer: ```suggestion
12. You are now connected to Box. Select **Finish** to close the window.
```
